src/PageContent/Settings/OrganizationDetails.js

```diff
@@ -2,16 +2,68 @@ import React, { Component } from 'react';
 import Map from '../../Map/Map.js';
 import { AccountType } from '../../Enums';
 import { accountsRef, donatingAgenciesRef } from '../../FirebaseConfig.js';
+import moment from 'moment-timezone';
 
 class OrganizationDetails extends Component {
 
     constructor(props) {
         super(props);
         this.state = {
-            isEditing: false
+            isEditing: false,
+            times: [],
         };
+        this.convertTimes = this.convertTimes.bind(this);
+        this.dayRow = this.dayRow.bind(this);
+        this.dayNames = ['Sun','Mon','Tue','Wed','Thur','Fri','Sat'];
+
+
+    }
+
+    dayRow(i, day) {
+
+        var checkboxName = day + 'Check';
+        var startName = day + 'Start';
+        var endName = day + 'End';
+
+        // populate default values if exists
+        var checked = false;
+        var startTime = '';
+        var endTime = '';
+        var availabilities = this.props.org.availabilities;
+        if (availabilities && availabilities[i]) {
+            checked = true;
+            startTime = availabilities[i].startTimestamp;
+            endTime = availabilities[i].endTimestamp;
+            console.log('startTime:', startTime)
+        }
+
+        return (
+            <div className="row" key={day}> 
+                <input type="checkbox" name={checkboxName} defaultChecked={checked} />
+                <div className="day">{day}</div>
+                {/* TODO: AM/PM UI */}
```

Comment:
What do you mean by AM/PM UI here? Is this a separate thing or did you end up implementing it?